src/components/TooltipIcon/__tests__/__snapshots__/index.tsx.snap

```diff
@@ -19,6 +19,7 @@ exports[`TooltipIcon renders with default props 1`] = `
     tabindex="0"
   >
     <svg
+      aria-label="help-circle-outline"
```

Comment:
Il serait plus sympa de passer la chaîne à la traduction, mais je ne sais pas évaluer si ça vaut vraiment le coup...

Reply:
Yes it was an option but it means we need to provide a i18n text just for the SVG label :/ not sure it is really worth it?

Reply:
@adriengibrat I will let it go as is for now. We can think about something new a bit later maybe if we feel the need.